pkg/modelzooserver/modelzooserver_test.go

```diff
@@ -109,7 +109,7 @@ func TestModelZooServer(t *testing.T) {
 		err = os.Chdir(dir)
 		a.NoError(err)
 
-		err = tar.TarGzDir(".", "modelrepo.tar.gz")
+		err = tar.GzDir(".", "modelrepo.tar.gz")
```

Comment:
Should change both function names: `ZipDir`, `UnzipDir`

Reply:
done